src/app/users/views/confirmNewUser.ejs

```diff
@@ -75,21 +75,34 @@
                     <label class="form-label" for="email-contents">Email contents</label>
                     <div class="email-editor">
                         <p>Dear <%=user.firstName%> <%=user.lastName%></p>
-                        <textarea class="form-control full-width" rows="8" name="email-contents" id="email-contents">You've received this email because you've been invited to join DfE Sign-in.</textarea>
-                        <p>Please follow the link at the bottom of this email and enter the code below so we can create your account.</p>
-                        <div class="static-text">
-                            <p>Your verification code is:</p>
-
-                            <p class="code">VERFICATION CODE</p>
-
-                            <p>To verify your email address and set up your account, please follow the link below: <em>LINK TO THE SERVICE </em></p>
-
-                            <p><em>If you received this email in error, please ignore it.</em></p>
-
-                            <p>Kind regards,</p>
-
-                            <p>DfE Sign-in</p>
-                        </div>
+                        <% if (locals.entra && locals.entra.enableEntraSignIn === true) { %>
```

Comment:
Can we change this line to match the flag for user registration 
`<% if (locals.entra && locals.entra.useEntraForAccountRegistration === true){ %>`